vignettes/convert.qmd

```diff
@@ -0,0 +1,346 @@
+
+The advantage of this format is that you can "update" it quickly. For example, if you first downloaded the data for March and April 2020, converted this period into `parquet` format, and then downloaded the data for May and June 2020, when you run the convertion function again, it will only convert the data for May and June 2020 and add it to the existing `parquet` files. So you will save time and will not have to wait for March and April 2020 to be converted again.
+
+### Convert to `Parquet` {#convert-to-parquet}
+
+Let us convert a few dates to `parquet` format:
+
+```{r}
+type <- "od"
+zones <- "distr"
+dates <- c(start = "2020-02-14", end = "2020-02-17")
+od_parquet <- spod_convert(type = type, zones = zones, dates = dates, save_format = "parquet")
+```
+
+If we now request additional dates that overlap with the already converted data like so and specifiy argument `overwrite = 'update'` we will update the existing `parquet` files with the new data:
+
+```{r}
+dates <- c(start = "2020-02-16", end = "2020-02-19")
+od_parquet <- spod_convert(type = type, zones = zones, dates = dates, save_format = "parquet", overwrite = 'update')
+```
+
+That is, 16 and 17 Feboruary will not be converted again. Only the new data, that was not converted (18 and 19 February) will be converted, and these will be added to the existing folder structure of`parquet` files stored at the default `save_path` location, which is `<data_dir>/clean_data/v1/tabular/parquet/od_distritos`. Alternatively, you can set any other save location by setting the `save_path` argument.
+
+```{r}
+#| echo: false
+# Test for v2 data
+dates <- c(start = "2023-02-14", end = "2023-02-17")
+od_parquet <- spod_convert(type = type, zones = zones, dates = dates, save_format = "parquet", save_path = file.path(tempdir(), "od_parquet"))
+```
+
+### Load the converted `Parquet` {#load-converted-parquet}
+
+Working with these `parquet` files is exactly the same as with `DuckDB` and `Arrow` files. Just like before, you can use the same helper function `spod_connect()` to connect to the `parquet` files:
+
+```{r}
+my_od_data_3 <- spod_connect(od_parquet)
+```
+
+Mind you though, because we have first converted the data for a period between 14 and 17 February 2020, and then converted the data for a period between 16 and 19 February 2020 into the save default location, the `od_parquet` contains the path to all this data, and therefore `my_od_data_3` will connect you to all data.
+
+You can check this like so:
+
+```{r}
+my_od_data_3 |> 
+  dplyr::distinct(date) |>
+  dplyr::arrange(date)
+```
+
+To prepare all origin-destination data v1 (2020-2021) for analysis over the whole period of data availability, please follow the steps below:
```

Comment:
I find this redundant or perhaps place it under the heading name 'Download all available data'